homework-4/script.js

```diff
@@ -0,0 +1,15 @@
+import users from "./users.json" assert { type: "json" };
+
+function getUserInfo(users) {
+  let phoneNumbersArray = [];
+  for (let i = 0; i < users.length; i++) {
+    let deleteMarkDollar = users[i].balance.slice(1);
+    let convertToNumber = Number(deleteMarkDollar.replace(",", ""));
+    if (convertToNumber > 2000) {
+      phoneNumbersArray.push(users[i].phone);
+    }
+  }
+  return phoneNumbersArray;
+}
```

Comment:
1. I would like you to make in a separate file helpers.js, a convertBalance function that would take a string and return a number
2. I would like you to use the Array.reduce method instead of the loop